alephium/.env.testnet

```diff
@@ -0,0 +1,8 @@
+INIT_SIGNERS=["D5f4d28216544037CB228af1D11D6b5B45067233"]
+INIT_CHAIN_ID=255
+
+INIT_GOV_CHAIN_ID=1
+INIT_GOV_CONTRACT=0000000000000000000000000000000000000000000000000000000000000004
```

Comment:
What is this?

Reply:
This is the placeholder governance contract id that needs to be consistent across all chains. When a governance action is received by smart contract on a chain (e.g. from CLI), the emitter address of the VAA needs to match with this contract id.

Reply:
@Lbqds Why the contract is postfixed with `4`?

Reply:
The governance emitter address is an off-chain address, we can set the `INIT_GOV_CONTRCT` to any 32-byte value. Wormhole uses the `0000000000000000000000000000000000000000000000000000000000000004` as the governance emitter address, and we didn't change it.

Reply:
Is it possible that `4` is the version of the gov contract?

Reply:
I guess not, from these transactions:

https://etherscan.io/tx/0x45677f558e3623dc3f9fd346a8f06ed15852aeab7686e5a9beefd8f166e0fe63
https://etherscan.io/tx/0x01b299d4b52d7df2504157717a895ec5bbe3277038c8918f379997e82c54cd47

we can see that the wormhole uses `0000000000000000000000000000000000000000000000000000000000000004` as the governance emitter address from the beginning.

Reply:
once the contract are deployed the version would not change, maybe there were some testnet deployments which use different emitter address.

Reply:
On the Goerli  testnet, the emitter address is `0000000000000000000000000000000000000000000000000000000000000040`, transactions:

https://goerli.etherscan.io/tx/0x328a36ef868fdc2ee054d8d737d2e748356c30c7f4b81aafb17bee2971c9212f
https://goerli.etherscan.io/tx/0x54707d36880eeb8faad7778006b4ceb828e0c72c87f94dbd24e364c5d21341d1